src/App.tsx

```diff
@@ -4,15 +4,16 @@ import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import { LoginPage } from './views/LoginPage';
 import { SampleDashboard } from './views/SampleDashboard';
 import { RegistrationLandingPage } from './components/RegistrationLandingPage';
+import { Home } from './components/views/Home/Home';
 
 export const App: FC = () => (
-
   <div>
+    <Home />
```

Comment:
Refaktoryzacja polega też na usuwaniu niepotrzebnych już elementów :)

Reply:
To akuratnie ja dodałem. Nie mamy żadnego widoku na '/', wydaje mi się, że powinno być cokolwiek, by nie prowadzić nigdy do pustej strony. Może powinniśmy dać tutaj coś defaultowego?

Reply:
Trzeba zrobić po prostu przekierowanie do logowania. (Taka moja pierwsza mysl).